CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 0.47.0-beta.1
+
+## Bug fixes
+* Use updateData instead of re-creating buffers for repopulated paint arrays ([#6853](https://github.com/mapbox/mapbox-gl-js/pull/6853))
+* Fix ScrollZoom handler setting tr.zoom = NaN ([#6924](https://github.com/mapbox/mapbox-gl-js/pull/6924))
+  - Failed to invert matrix error ([#6486](https://github.com/mapbox/mapbox-gl-js/issues/6486), fixed by [#6924](https://github.com/mapbox/mapbox-gl-js/pull/6924))
```

Comment:
Did you mean to indent these two items?

Reply:
This is `changelog-draft`'s behavior -- I assume because the PR closed these two issues.